README.md

```diff
@@ -1,13 +1,13 @@
 # Babysitter Finder ![Status badge](https://img.shields.io/badge/status-in%20progress-yellow)
 
 
-Babysitter finder is a project that offers you the babysitter service, through science and astheroids study.
+Babysitter finder is a project that offers you the babysitter service, through science and asteroids study.
```

Comment:
Let's rewrite this @martin-stepwolf if you can help me because is not about asteroids the project

Reply:
Yes, I already realized it, it looks like the README.md of NeoBit.